api-reference/beta/api/directory_post_deleteditems.md

```diff
@@ -0,0 +1,74 @@
+# Restore deletedItem
+
+Restores a [deleted item](../resources/directory.md).
+
+> **Important**: Deleted groups in the Recycle bin will be permanently deleted after 30 days.
```

Comment:
I called this out since you might try to restore a soft delete after 30 days. This is also mentioned in the resource topic.